javaagent-tooling/build.gradle.kts

```diff
@@ -113,6 +113,14 @@ testing {
         compileOnly("com.google.code.findbugs:annotations")
       }
     }
+
+    val testConfigFile by registering(JvmTestSuite::class) {
+      dependencies {
+        implementation(project(":javaagent-tooling"))
+        // requires mockito-inline
+        implementation("uk.org.webcompere:system-stubs-jupiter")
```

Comment:
did the system stub extension break something in other test to make you move this test into separate suite?